src/utils/dom.ts

```diff
@@ -1,3 +1,8 @@
+import { Theme } from "../components/interfaces";
+import { CSS_UTILITY } from "./resources";
+
+export const themeNameCSSVariable = "--calcite-theme-name";
```

Comment:
do you need this still?

Reply:
No but I left it there incase there is a need. I can remove it.